students/ltrisolini/Lesson02/series.py

```diff
@@ -0,0 +1,49 @@
+def fibonacci(n):
+    c = 1
+    a, b = 0, c
+    if n == 0:
+        return 0
+    for i in range(n-1):
+        a, b = b, a + b
+    return b
+
+
+def lucas(n):
+    c = 2
+    a, b = 0, c
+    if n == 0:
+        return 0
+    for i in range(n-1):
+        a, b = b, a + b
+    return b
+
+#printing fibonacci results
+print ("Fibonacci series:")
+print fibonacci(0)
+print fibonacci(1)
+print fibonacci(2)
+print fibonacci(3)
+print fibonacci(4)
+print fibonacci(5)
+print fibonacci(6)
+print fibonacci(7)
+print fibonacci(8)
+print fibonacci(9)
+print fibonacci(10)
```

Comment:
it is best to use `assert` statements so that you can actually test it instead of visually observing output, example:
```
assert fibonacci(9) == <expected_value>

```